src/server/Document/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { renderToString, renderToStaticMarkup } from 'react-dom/server';
+import { ServerStyleSheet } from 'styled-components';
+import { Helmet } from 'react-helmet';
+import { ServerApp } from '../../app/containers/App';
+
+import { getStyleTag } from '../styles';
+import getAssetsArray from '../assets';
+import Document from './component';
```

Comment:
Can I be a huge pain and ask for this to be `import DocumentComponent` please?, Document inside Document isnt v pretty imo

Reply:
Change has been made.